src/codes/june_challenge/day1/InvertBinaryTree.js

```diff
@@ -0,0 +1,14 @@
+var invertTree = function(root) {
+    if(!root) return root;
+    let arr = [];
+    arr.unshift(root);
+    while (arr.length != 0) {
+        let cur = arr.shift();
+        let temp = cur.left;
+        cur.left = cur.right;
+        cur.right = temp;
```

Comment:
You can modify the swapping logic with this `[cur.left, cur.right] = [cur.right, cur.left];`